BepInEx.Core/Configuration/ConfigFile.cs

```diff
@@ -46,21 +48,6 @@ public ConfigFile(string configPath, bool saveOnInit, BepInPlugin ownerMetadata)
 
     private Dictionary<ConfigDefinition, string> OrphanedEntries { get; } = new();
```

Comment:
There needs to be a way to get these without Binding, it's been a fairly commonly requested feature for setting migration and dynamic settings.

Reply:
reverted to public with an xmldoc